client/components/restaurant-list.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Rating from '../lib/rating.js';
+import hotTimes from '../lib/hot-times.js';
+
+function Restaurant(props) {
+  const style = {
+    backgroundImage: `url(${props.info.image_url})`,
+    backgroundPosition: 'center',
+    backgroundSize: 'cover',
+    backgroundRepeat: 'no-repeat',
+    borderRadius: '5px',
+    boxShadow: '2px 2px 2px rgb(110, 110, 110)'
+  };
+  return (
+    <>
+      <div className="image-container" style={style}/>
```

Comment:
What's wrong with using an `<img>`? `object-fit` will help if you are worried about the image warping.